src/chart/options/options.h

```diff
@@ -29,14 +29,22 @@ namespace Vizzu::Gen
 class Options
 {
 public:
+	enum class LegendId { color, lightness, size };
```

Comment:
Can we add some protection to ensure that the values are the same as in ChannelId?

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/399/commits/040dd47d42ee003b05af4860699a3046eb3e01f5